cmd/provisioner-localpv/app/provisioner_blockdevice.go

```diff
@@ -41,13 +41,23 @@ func (p *Provisioner) ProvisionBlockDevice(ctx context.Context, opts pvControlle
 	stgType := volumeConfig.GetStorageType()
 	fsType := volumeConfig.GetFSType()
 
+	bdTagValue := volumeConfig.GetBDTagValue()
+	if len(bdTagValue) > 0 {
+		klog.Infof("The 'BlockDeviceTag' cas.openebs.io/config key has been deprecated in favor of the 'BlockDeviceSelectors' key.\nSample config:\n" +
```

Comment:
In this sample config, can you change the example so that the example shows how to use openebs.io/block-device-tag with BlockDeviceSelectors.
For example, something like this...
```go
			"\t\t" + "- name: BlockDeviceSelectors\n" +
			"\t\t" + "  data:\n" +
			"\t\t" + "    \"openebs.io/block-device-tag\": \"mongo\"\n")
```

Reply:
Can you also put an appropriate error message? It is empty.

Reply:
Sure.